### DISCUSSION TITLE
Possible to make it compatible with older node versions?

### DISCUSSION THREAD

**Discussion by RafaelCENG** [wuba/react-native-echarts#68] [Q&A] [answered]
error @wuba/react-native-echarts@1.1.2: The engine "node" is incompatible with this module. Expected version ">= 16.0.0". Got "14.21.3"

Our project is based on node 14. Is there a workaround to use react-native-echarts?

**Comment by zhiqingchen** [comment, marked as answer]
Yes, try 1.1.3-alpha.0, the issue will be fixed in the release version later.
